packages/taquito/src/constants.ts

```diff
@@ -34,6 +34,7 @@ export enum Protocols {
   PtJakart2 = 'PtJakart2xVj7pYXJBXrqHgd82rdkLey5ZeeGwDgPp9rhQUbSqY',
   PtKathman = 'PtKathmankSpLLDALzWw7CGD2j2MtyveTwboEYokqUCP4a1LxMg',
   PtLimaPtL = 'PtLimaPtLMwfNinJi9rCfDPWea8dFgTZ1MeJ9f1m2SRic6ayiwW',
+  PtGhostnet = 'PtLimaPtLMwfNinJi9rCfDPWea8dFgTZ1MeJ9f1m2SRic6ayiwW',
```

Comment:
Let's not do this since Ghostnet isn't a protocol but just a network. Change the `protocol` parameter for Ghostnet in `config.ts` to be `PtLimaPtL` instead.

Reply:
Done and it fixes it thanks